crates/pathfinder/src/ethereum/state_update/state_root.rs

```diff
@@ -5,19 +5,21 @@ pub type StateRootFetcher = LogFetcher<StateUpdateLog>;
 
 #[cfg(test)]
 mod tests {
-    use crate::{core::StarknetBlockNumber, ethereum::test::create_test_websocket};
     use assert_matches::assert_matches;
     use pretty_assertions::assert_eq;
-    use web3::Web3;
+
+    use crate::{
+        core::StarknetBlockNumber,
+        ethereum::{test::create_test_transport, Chain},
+    };
 
     use super::*;
 
     #[tokio::test]
     async fn genesis() {
         // The first state root retrieved should be the genesis event,
         // with a sequence number of 0.
-        let ws = create_test_websocket().await;
-        let ws = Web3::new(ws);
+        let ws = create_test_transport(Chain::Goerli).await;
```

Comment:
Absolutely not an issue - maybe you wanna rename those `ws` instances to `http` or event `t` so as to avoid renaming next time..?